lib/utils/isStandardSyntaxSelector.js

```diff
@@ -29,6 +34,16 @@ module.exports = function (selector) {
 		return false;
 	}
 
+	// Less non-outputting mixin definition (e.g. .mixin() {})
+	if (selector.endsWith(')') && !selector.includes(':')) {
+		return false;
+	}
+
+	// Less Parametric mixins (e.g. .mixin(@variable: x) {})
+	if (/\(@.*\)$/.test(selector)) {
+		return false;
+	}
+
```

Comment:
@immitsu Thank you for addressing my reviews! Additionally, can you add test cases for the logic added to `isStandardSyntaxSelector()`, please?

- `SCSS nested properties`
- `Less non-outputting mixin definition (e.g. .mixin() {})`
- `Less Parametric mixins (e.g. .mixin(@variable: x) {})`

Reply:
@ybiquitous Done!